gitops-charts/humio-helm-charts/CHANGELOG.md

```diff
@@ -0,0 +1,203 @@
+# Changelog
```

Comment:
what does changelog means here? Does the helm chart from open source?

Reply:
I've removed the changelog file to avoid confusion.